### PR TITLE
Added FliteTest Model Icons

### PR DESCRIPTION
**Pull request by RCdiy**
Renamed WW2 > Scale
New FliteTest model icons with permission to use.
http://forum.flitetest.com/showthread.php?34403-Frsky-Taranis-Model-Icons

E-mail received
Hi,

Thanks for your contact on the Flitetest forum!

Attached is a zip file with the Flitetest icons stored in appropriate folders to match with the opentx folder structure. I did not change the file names to all lower case, like all of their existing names seem to be. I wonder if that will be an issue...

Jeff

### CODE REVIEW

File: update-indexes.py [hunk 1]
```diff
@@ -1,53 +1,53 @@
-#! python3.4
+#! /usr/bin/python
+# #! python3.4
```
Comment (schwabe) [inline update-indexes.py]: Don't leave the old stuff in it. And better use python3 instead of python

**Comment by schwabe** [comment]
I see a lot of other files modified or moved into different directories in this commit. Was that an accident or intended?

File: update-indexes.py (hunk 1 above)
Comment (RCdiy) [inline update-indexes.py]: Does not run on my mac...

**Comment by RCdiy** [comment]
Added a number or icons.
Renamed WW2 > Scale

**Comment by schwabe** [comment]
Could you document in this PR where the new model images come from and they have the proper license to include them?

**Comment by RCdiy** [comment]
I've e-mailed flitetest and the contributor; Waiting for a response.

**Comment by RCdiy** [comment]
Could you point me to similar permissions received for the other icons already there? THis is so that I can better understand the wording you are looking for.

FliteTest is by its nature opensource with its plane plans. The icons being submitted are already posted publicly and the icon author sen me the icons by e-mail to be posted on the OpenTX website.

Posting these icons is not against the spirit of FliteTest and at worst we will receive a simple take down request.

**Comment by schwabe** [comment]
To be honest I am not sure how it was handled with previous. But I just asked this as a safety thing. Rights can be tricky and better safe than sorry. 

Just some statement that these files can be included with OpenTX and its license (GPL) from the author would be nice. When author directly do pull request/send stuff to the project that is an implicit grant of this but if goes via a 3rd party here, we need that explicitly to be on safe side.

Maybe I am overreacting a bit but too may projects had problem because of this (e.g. Cleanflight configurator and its icons)

**Comment by RCdiy** [comment]
The icon author sent me the icons to be posted. Maybe when I'm at Flitefest I can get permission from Flitetest as well. Efforts to reach them online have not been successful so far.

**Comment by RCdiy** [comment]
Permission received @schwabe
http://forum.flitetest.com/showthread.php?35657-Copyright-Taranis-Model-Icons&p=355067#post355067

**Comment by RCdiy** [comment]
To whom it may concern,

Flite Test gives permission to use FT icons as Frsky Taranis Model Icons for personal use.

Stefan Ostergren
Flite Test

**Comment by schwabe** [comment]
Thanks!

**Comment by RCdiy** [comment]
@schwabe apologies I missed your response:) 
Will this pull request be accepted?

**Comment by RCdiy** [comment]
@schwabe @bsongis Is there anything else to be done to get this pull request accepted?

**Comment by bsongis** [comment]
Sorry it's just me, I have been busy with a lot of other things during latest weeks, I plan to return to business next week :)

**Pull request merged**